skymp5-server/cpp/CMakeLists.txt

```diff
@@ -188,6 +188,9 @@ foreach(target ${VCPKG_DEPENDENT})
   endif()
 endforeach()
 
-find_package(ChakraCore REQUIRED)
+find_path(ChakraCore_INCLUDE_DIR NAMES ChakraCore.h)
+find_library(ChakraCore_LIBRARY_Debug NAMES ChakraCore)
+string(REPLACE "/debug/lib/" "/lib/" ChakraCore_LIBRARY_Release "${ChakraCore_LIBRARY_Debug}")
+
 # don't want to link against MpClientPlugin for example..
-target_link_libraries(server_guest_lib PUBLIC ChakraCore::ChakraCore)
+target_link_libraries(server_guest_lib PUBLIC "$<IF:$<CONFIG:Debug>,${ChakraCore_LIBRARY_Debug},${ChakraCore_LIBRARY_Release}>")
```

Comment:
Does this ChakraCore stuff relate to SKSE build?

Reply:
No, it doesn't. This PR changes the way we link SKSE into our target. Now it's taken from vcpkg. We also fix up paths to ensure they are different on different configs (Release/Debug). So, this PR also fixes the configs problem for all dependencies we have.